src/System Application/App/Azure File Services API/src/AFSFileClient.Codeunit.al

```diff
@@ -44,6 +44,17 @@ codeunit 8950 "AFS File Client"
         AFSFileClientImpl.Initialize(StorageAccount, FileShare, '', Authorization, APIVersion);
     end;
 
+    /// <summary>
+    /// The base URL to use when constructing the final URI.
+    /// If not set, the base URL is https://%1.file.core.windows.net where %1 is the storage account name.
+    /// </summary>
+    /// <remarks>Use %1 as a placeholder for the storage account name.</remarks>
+    /// <param name="BaseUrl">A valid URL string</param>
+    procedure SetBaseUrl(BaseUrl: Text)
```

Comment:
Isn't it better to have an override of `Initialize` that accepts a base URL?

Reply:
Designed it this way to maintain the existing logic in the ABS module. Let me know if you still think it's better to add an override, and if so, whether we should also add it to the `Initialize` method in the ABS module.

Reply:
Let's keep it as it is to align with the ABS module.